pylint/config/callback_actions.py

```diff
@@ -398,13 +398,13 @@ def __call__(
     ) -> None:
         assert isinstance(values, (tuple, list))
         msgids = utils._check_csv(values[0])
-        try:
-            for msgid in msgids:
+        for msgid in msgids:
+            try:
                 self.linter.enable(msgid)
-        except exceptions.UnknownMessageError:
-            # pylint: disable-next=fixme
-            # TODO: Optparse: Raise an informational warning here
-            pass
+            except exceptions.UnknownMessageError:
+                # pylint: disable-next=fixme
+                # TODO: Optparse: Raise an informational warning here
+                pass
```

Comment:
What about doing it right now ? I don't like this silent fail, in fact we had a more damaging silent fail before and we've been lucky that Marc signaled the issue.

Reply:
Sure! Drafting this then!